doc-Managing_Infrastructure_and_Inventory/topics/Instances.adoc

```diff
@@ -1,29 +1,32 @@
 [[instances]]
-= Instances
+= Instances and Images
+*Instances* and *Images* 
 
-The *Instance* container combined with the ability to analyze information inside each instance provides in-depth information across the cloud environment.
-This rich set of information enables {product-title} users to improve problem resolution times and effectively manage instances in their cloud environment.
+*Images* are the static templates containing the software configuration from which you provision a running *Instance* - a virtual machine, with which you can interact - on your cloud provider. 
 
-The *Instances* pages display all instances the server discovered from your cloud providers.
-The *Instances* taskbar is a menu driven set of buttons that provide access to functions related to instances.
+The *Instance* and *Images* containers, combined with the ability to analyze information inside each instance or image, provides in-depth information across the cloud environment.
+This rich set of information enables {product-title} users to improve problem resolution times and effectively manage instances and images in their cloud environment.
 
+The *Instances* and *Images* pages display all instances and images the server discovered from your cloud providers.
+The taskbar on each page is a menu driven set of buttons that provide access to functions related to instances and images.
 
-image:4222.png[]
+
+image:image_features.png[]
 
 . History button
 . Refresh screen button
 . Taskbar
-. Name search bar/Advanced Search button
-. View buttons
 . Download buttons
-. Navigation bar
+. View buttons
+. Name search bar/Advanced Search button
 . Sort dropdown
-. Main area in Grid View
+. Navigation bar
+. Main area in List View
 . Cloud/Filter Navigation
 
 Console uses *Virtual Thumbnails* to describe instances and images.
 Each thumbnail contains four quadrants by default.
-This allows you to glance at an instance for a quick view of its contents.
+This allows you to glance at an instance or image for a quick view of its contents.
 
 
 image:3393.png[]
```

Comment:
Hey Chris, I think it used to be in one of the miq repos, though it might now be residing under patternfly - sorry, I'm not sure either. As discussed, we can look into this one later, the rest of the changes look good.